src/qt/stattext.cpp

```diff
@@ -55,6 +55,34 @@ bool wxStaticText::Create(wxWindow *parent,
     m_qtLabel->setBuddy( m_qtLabel );
     m_qtLabel->setTextInteractionFlags( Qt::NoTextInteraction );
 
+    Qt::AlignmentFlag qt_alignment = Qt::AlignAbsolute;
+
+    switch (style & wxALIGN_MASK)
+    {
+        case wxALIGN_RIGHT:
+            qt_alignment = Qt::AlignRight;
+            break;
+        case wxALIGN_LEFT:
+            qt_alignment = Qt::AlignLeft;
+            break;
+        case wxALIGN_CENTER:
+            qt_alignment = Qt::AlignCenter;
+            break;
+        case wxALIGN_CENTER_HORIZONTAL:
+            qt_alignment = Qt::AlignHCenter;
+            break;
+        case wxALIGN_CENTER_VERTICAL:
```

Comment:
Is vertical alignment actually supported by this control in Qt? None of the other ports supports it, so I'd just remove this case and the one below. If you do want to support it, you need to do something about `wxALIGN_TOP` too (see the comment for the `switch` above).